README.md

```diff
@@ -152,11 +152,11 @@ The shared storage worklet invocation methods (`addModule`, `createWorklet`, and
     *   Options include:
         *   `withLock`: acquire a lock on the designated resource before executing. See [Locking for modifier methods](#locking-for-modifier-methods) for details.
 *   `window.sharedStorage.batchUpdate(methods, options)`
-    *   Execute `methods` in order.
-    *   `methods` is an array of method objects defining the operations to perform. Each object must be one of the following types: `SharedStorageSetMethod`, `SharedStorageAppendMethod`, `SharedStorageDeleteMethod`, or `SharedStorageClearMethod`. Each method object's constructor accepts the same parameters as the corresponding individual method (e.g., `set`, `append`, `delete`, `clear`).
+    *   Execute `methods` in order. All updates within a `batchUpdate()` call are executed as a single unit of work.
+    *   `methods` is an array of method objects defining the operations to perform. Each object must be one of the following types: `SharedStorageSetMethod`, `SharedStorageAppendMethod`, `SharedStorageDeleteMethod`, or `SharedStorageClearMethod`. Each method object's constructor accepts the same parameters as the corresponding individual method (e.g., `set`, `append`, `delete`, `clear`). Note that the `withLock` option within these methods will be ignored (for transactional behavior).
```

Comment:
Just to make it super clear, I'd specify that the withLock options within the individual methods will be ignored, but the lock for the batchUpdate() method itself will be acquired.

Reply:
Done.